templates/definition/tariff/energy-charts-api.yaml

```diff
@@ -0,0 +1,18 @@
+template: energy-charts-api
+products:
+  - brand: Fraunhofer ISE
+params:
+  - name: bzn
+    type: string
+    required: true
+    help:
+      de: "Gebotszonen: AT (Austria), BE (Belgium), CH (Switzerland), CZ (Czech Republic), DE-LU (Germany, Luxembourg), DE-AT-LU (Germany, Austria, Luxembourg), DK1 (Denmark 1), DK2 (Denmark 2), FR (France), HU (Hungary), IT-NORTH (Italy North), NL (Netherlands), NO2 (Norway 2), PL (Poland), SE4 (Sweden 4), SI (Slovenia)"
+      en: "Bidding zones: DE-LU  AT (Austria), BE (Belgium), CH (Switzerland), CZ (Czech Republic), DE-LU (Germany, Luxembourg), DE-AT-LU (Germany, Austria, Luxembourg), DK1 (Denmark 1), DK2 (Denmark 2), FR (France), HU (Hungary), IT-NORTH (Italy North), NL (Netherlands), NO2 (Norway 2), PL (Poland), SE4 (Sweden 4), SI (Slovenia)"
```

Comment:
Das ist ein ganz schön langer Hilfetext :D

Wir haben das Konzept der `validvalues`. Würde vorschlagen, dass wir das hier verwenden. Dann natürlich nur die Kürzel. Gggf. dann noch mit einem Link im Description-Block zur Erklärung vom ISE.

In der UI können wir da dann ein schönes Dropdown daraus generieren und die Übersetzung der Zonennamen über Weblate machen.

Siehe: https://github.com/evcc-io/evcc/blob/236f2f8f3de6857494adaf7eb3e19c0c832bda24/templates/definition/meter/kostal-plenticore.yaml#L31-L32

Reply:
Habe ich mir schon fast gedacht-- ich passe es an ;)